app/models/grouping.rb

```diff
@@ -756,14 +749,14 @@ def student_test_run_in_progress?
 
   def access_repo
     group.access_repo do |repo|
-      add_starter_files(repo)
+      add_assignment_folder(repo)
       yield repo
```

Comment:
Please update this line to yield the repo if a block is given (this should allow you to call `grouping.access_repo` instead of `grouping.access_repo {}`).